packages/kbn-cell-actions/src/types.ts

```diff
@@ -82,6 +83,8 @@ export interface CellActionsProps {
   metadata?: Metadata;
 
   className?: string;
+
+  children?: React.ReactNode;
```

Comment:
This `CellActionsProps` type is not only used by components, the `useDataGridColumnsCellActions` hook generated its parameters types based on this definition. 
To not overcomplicate things we can just omit it by adding: `| 'children' ` here:

https://github.com/elastic/kibana/blob/fa936db23be141a2c69952dfe4e6b201d4a4a980/packages/kbn-cell-actions/src/hooks/use_data_grid_column_cell_actions.tsx#L25-L26

For the rest, we should be good.

Reply:
I am rereading this comment, and it does not make sense 😂. we are already `Pick`ing not `Omit`ting. So we are not taking the `children`, we are good.